doc/README.md

```diff
@@ -1,2 +1,8 @@
-Require `Sphinx`  
-Use `make builder` where `builer` can be `HTML`, `Latex`, ... 
\ No newline at end of file
+## Require `Sphinx`  
```

Comment:
remove this `README.md` and add a section in the root's readme about how to use tox to generate doc.

And remove the makefile in this dir.

Reply:
done